CSS/Question-01.css

```diff
@@ -0,0 +1,14 @@
+.content__title {
+    font-weight: 700;
+    font-size: 36px;
+}
+
+.content__subtitle {
+    font-size: 24px;
+}
+
+.btn__question-01 {
+    display: flex;
+    justify-content: center;
+    font-size: 16px;
```

Comment:
`font-size` property is already 16px since it is the browser default, please remove this line